int32 NUM_BYTES = 6
uint8[6] motor_packet
uint8 HEADER = 255 # 255 = 0xFF
uint8 MAGIC_NUMBER = 170 # 170 = 0xAA